### ISSUE TITLE
404 for operators.js

### ISSUE THREAD

**Issue by acrisci** [spring-projects/spring-flo#14]
Hi,

After a fresh clone and `npm install`, starting the project with `npm run start` and going to the page gives me this 404 error.

```
(index):19 Error: (SystemJS) XHR error (404 Not Found) loading http://localhost:3000/node_modules/rxjs/operators.js
	Error: XHR error (404 Not Found) loading http://localhost:3000/node_modules/rxjs/operators.js
	    at XMLHttpRequest.wrapFn (http://localhost:3000/node_modules/zone.js/dist/zone.js:1166:39)
	    at ZoneDelegate.invokeTask (http://localhost:3000/node_modules/zone.js/dist/zone.js:425:31)
	    at Zone.runTask (http://localhost:3000/node_modules/zone.js/dist/zone.js:192:47)
	    at ZoneTask.invokeTask [as invoke] (http://localhost:3000/node_modules/zone.js/dist/zone.js:499:34)
	    at invokeTask (http://localhost:3000/node_modules/zone.js/dist/zone.js:1540:14)
	    at XMLHttpRequest.globalZoneAwareCallback (http://localhost:3000/node_modules/zone.js/dist/zone.js:1566:17)
	Error loading http://localhost:3000/node_modules/rxjs/operators.js as "./operators" from http://localhost:3000/node_modules/rxjs/Rx.js
	    at XMLHttpRequest.wrapFn (http://localhost:3000/node_modules/zone.js/dist/zone.js:1166:39)
	    at ZoneDelegate.invokeTask (http://localhost:3000/node_modules/zone.js/dist/zone.js:425:31)
	    at Zone.runTask (http://localhost:3000/node_modules/zone.js/dist/zone.js:192:47)
	    at ZoneTask.invokeTask [as invoke] (http://localhost:3000/node_modules/zone.js/dist/zone.js:499:34)
	    at invokeTask (http://localhost:3000/node_modules/zone.js/dist/zone.js:1540:14)
	    at XMLHttpRequest.globalZoneAwareCallback (http://localhost:3000/node_modules/zone.js/dist/zone.js:1566:17)
	Error loading http://localhost:3000/node_modules/rxjs/operators.js as "./operators" from http://localhost:3000/node_modules/rxjs/Rx.js
```

**Comment by acrisci** [comment]
https://github.com/ReactiveX/rxjs/issues/2971

**Comment by acrisci** [comment]
workaround:

```
echo 'require('./operators/index.js');' > node_modules/rxjs/operators.js
```

**Comment by BoykoAlex** [comment]
@acrisci I have decided to fix the version for `rxjs` at 5.4.3. It's used for sample only anyway.
Fixed with #c8fadb1ba7636cc2af40286a33a8722526f44d9b
